packages/es-indexer/src/indexers/TrackIndexer.ts

```diff
@@ -95,7 +95,8 @@ export class TrackIndexer extends BaseIndexer<TrackDoc> {
     -- etl tracks
     select 
       tracks.*,
-      case when tracks.premium_conditions->>'usdc_purchase'
+      case when tracks.stream_conditions->>'usdc_purchase'
+        or tracks.download_conditions->>'usdc_purchase'
```

Comment:
good question. not sure tbh

Reply:
i think this can be removed, cuz with the new specs, `tracks.download_conditions->>'usdc_purchase'` if and only if `tracks.stream_conditions->>'usdc_purchase'` is true